Automattic/woocommerce-payments
# Use npm version command to update the package.json files in the release workflow

Fixes #5708 

#### Changes proposed in this Pull Request

- Use `npm version` command to update the version in `package,.json` and `package-lock.json` files

This avoids manually updating the value in `package.json` and then running `npm install` to update the lock file.
This is a quick win to bring down the whole job's duration from ~1.5min to ~30s

#### Testing instructions

* Run the `release-pr.yml` workflow and check that the PR raised includes the version changes in both `package.json` and `package-lock.json` files

The test run I did raised this PR: https://github.com/Automattic/woocommerce-payments/pull/5932

-------------------


**Post merge**
